media-video/snapshot/snapshot-48.0.1.ebuild

```
@@ -0,0 +1,321 @@
+	nix@0.29.0
+	nu-ansi-term@0.46.0
+	num-integer@0.1.46
+	num-rational@0.4.2
+	num-traits@0.2.19
+	objc-foundation@0.1.1
+	objc@0.2.7
+	objc_id@0.1.1
+	once_cell@1.21.1
+	option-operations@0.5.0
+	ordered-stream@0.2.0
+	overload@0.1.1
+	pango-sys@0.20.9
+	pango@0.20.9
+	parking@2.2.1
+	paste@1.0.15
+	percent-encoding@2.3.1
+	pin-project-lite@0.2.16
+	pin-utils@0.1.0
+	piper@0.2.4
+	pkg-config@0.3.32
+	polling@3.7.4
+	portable-atomic-util@0.2.4
+	portable-atomic@1.11.0
+	ppv-lite86@0.2.21
+	proc-macro-crate@3.3.0
+	proc-macro2@1.0.94
+	quote@1.0.40
+	rand@0.9.0
+	rand_chacha@0.9.0
+	rand_core@0.9.3
+	regex-automata@0.4.9
+	regex-syntax@0.8.5
+	regex@1.11.1
+	rmp-serde@1.3.0
+	rmp@0.8.14
+	rqrr@0.9.0
+	rustc_version@0.4.1
+	rustix@0.38.44
+	rustix@1.0.2
+	semver@1.0.26
+	serde@1.0.219
+	serde_derive@1.0.219
+	serde_repr@0.1.20
+	serde_spanned@0.6.8
+	sharded-slab@0.1.7
+	shlex@1.3.0
+	signal-hook-registry@1.4.2
+	slab@0.4.9
+	smallvec@1.14.0
+	stable_deref_trait@1.2.0
+	static_assertions@1.1.0
+	syn@2.0.100
+	synstructure@0.13.1
+	system-deps@7.0.3
+	target-lexicon@0.12.16
+	temp-dir@0.1.14
+	tempfile@3.19.0
+	thiserror-impl@2.0.12
+	thiserror@2.0.12
+	thread_local@1.1.8
+	tinystr@0.7.6
+	toml@0.8.20
+	toml_datetime@0.6.8
+	toml_edit@0.22.24
+	tracing-attributes@0.1.28
+	tracing-core@0.1.33
+	tracing-log@0.2.0
+	tracing-subscriber@0.3.19
+	tracing@0.1.41
+	uds_windows@1.1.0
+	unicode-ident@1.0.18
+	url@2.5.4
+	utf16_iter@1.0.5
+	utf8_iter@1.0.4
+	valuable@0.1.1
+	version-compare@0.2.0
+	wasi@0.13.3+wasi-0.2.2
+	winapi-i686-pc-windows-gnu@0.4.0
+	winapi-x86_64-pc-windows-gnu@0.4.0
+	winapi@0.3.9
+	windows-sys@0.59.0
+	windows-targets@0.52.6
+	windows_aarch64_gnullvm@0.52.6
+	windows_aarch64_msvc@0.52.6
+	windows_i686_gnu@0.52.6
+	windows_i686_gnullvm@0.52.6
+	windows_i686_msvc@0.52.6
+	windows_x86_64_gnu@0.52.6
+	windows_x86_64_gnullvm@0.52.6
+	windows_x86_64_msvc@0.52.6
+	winnow@0.7.4
+	wit-bindgen-rt@0.33.0
+	write16@1.0.0
+	writeable@0.5.5
+	xdg-home@1.3.0
+	yeslogic-fontconfig-sys@6.0.0
+	yoke-derive@0.7.5
+	yoke@0.7.5
+	zbus@5.5.0
+	zbus_macros@5.5.0
+	zbus_names@4.2.0
+	zerocopy-derive@0.8.23
+	zerocopy@0.8.23
+	zerofrom-derive@0.1.6
+	zerofrom@0.1.6
+	zerovec-derive@0.10.3
+	zerovec@0.10.4
+	zvariant@5.4.0
+	zvariant_derive@5.4.0
+	zvariant_utils@3.2.0
+"
+
+RUST_MIN_VER="1.82"
+
+inherit cargo gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="GNOME Camera application written in Rust"
+HOMEPAGE="https://apps.gnome.org/Snapshot"
+SRC_URI+=" ${CARGO_CRATE_URIS}"
+
+LICENSE="GPL-3+"
+LICENSE+="
+	Apache-2.0 Apache-2.0-with-LLVM-exceptions ISC MIT Unicode-3.0 ZLIB
+	|| ( LGPL-2.1+ MPL-2.0 )
+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="X wayland test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=gui-libs/libadwaita-1.7_alpha:1
+	>=dev-libs/glib-2.75.0:2
+	>=gui-libs/gtk-4.16.0:4[X?,wayland?]
+	>=media-libs/gstreamer-1.20.0
+	>=media-libs/gst-plugins-bad-1.20.0
+	>=media-libs/gst-plugins-base-1.20.0
+	media-plugins/gst-plugin-gtk4
+	media-video/pipewire
```

given README.md with gstreamer-pipewire
```suggestion
	media-video/pipewire[gstreamer]
```